utils/remote_control/navigator_keyboard_control/remote_control_lib/remote_control_lib.py

```diff
@@ -120,6 +122,22 @@ def station_hold(self, *args, **kwargs):
             parameters={'location': self.name}
         )
 
+    @_timeout_check
+    def deploy_thrusters(self, *args, **kwargs):
+        def cb(terminal_state, result):
+            # TODO(ironmig): check that it was actually succesful
+            rospy.loginfo('Thrusters deployed!')
+
+        self.task_client.run_task('DeployThrusters', done_cb=cb)
+
+    @_timeout_check
+    def retract_thrusters(self, *args, **kwargs):
+        def cb(terminal_state, result):
+            # TODO(ironmig): check that it was actually succesful
```

Comment:
Was this a TODO before merge or after? Or is this a reminder to do a visual check of the thrusters after deploy/retract?

Reply:
Good catch, added the check